modules/database/database.bal

```diff
@@ -461,11 +461,19 @@ public isolated transactional function getExperimentCount() returns int|error {
 #
 # + 'limit - The maximum number of experiments fetched in one call (default: `100`)
 # + offset - The offset applied to the sql query (default: `0`)
+# + 'ascending - 1 for asc sort, 0 for desc sort by step sequence
 # + return - The list of experiments or the encountered error
-public isolated transactional function getExperiments(int 'limit = 100, int offset = 0) returns ExperimentFull[]|error {
-    stream<ExperimentFull, sql:Error?> experiments = experimentDB->query(
-        `SELECT experimentId, name, description FROM Experiment ORDER BY name ASC LIMIT ${'limit} OFFSET ${offset};`
-    );
+public isolated transactional function getExperiments(int 'limit = 100, int offset = 0, int? 'ascending = 1) returns ExperimentFull[]|error {
+    stream<ExperimentFull, sql:Error?> experiments;
+    if ('ascending != () && 'ascending == 1) || 'ascending == () {
+        experiments = experimentDB->query(
+            `SELECT experimentId, name, description FROM Experiment ORDER BY name ASC LIMIT ${'limit} OFFSET ${offset};`
```

Comment:
Test if sort order can be inserted similar to limit and offset (to deduplicate code better)